src/python/pants/backend/python/rules/pytest_runner.py

```diff
@@ -251,9 +245,10 @@ async def run_python_test(
     if use_coverage:
         output_files.append(".coverage")
 
-    # We explicitly add the CWD to the PEX runtime sys.path. Some pytest plugins
```

Comment:
I'm not sure the django example will still work, but if not I'll fix it, so this is fine for now.

Reply:
Okay, thank you! If it turns out this breaks, it would be good to add a unit test.